WORKERJS.md

```diff
@@ -0,0 +1,53 @@
+## Locating `worker.js` in the Google Cloud Function environment
+
+First, prepare an `index.js` that is valid, but has a runtime error
```

Comment:
I like the punctuation you added in this file, but I think it could do with a little more. Maybe a colon after this line as well as 23 and 27?

Reply:
fixed up line 12 instead of 23, right?

``` diff
--- a/WORKERJS.md
+++ b/WORKERJS.md
@@ -1,6 +1,6 @@
 ## Locating `worker.js` in the Google Cloud Function environment
 
-First, prepare an `index.js` that is valid, but has a runtime error
+First, prepare an `index.js` that is valid, but has a runtime error:
 
 > ``` js
 > exports.helloWorld = function helloWorld (req, res) {
@@ -10,7 +10,7 @@ exports.helloWorld = function helloWorld (req, res) {
 
 Next, deploy it, e.g. with `gcloud beta functions deploy`
 
-Finally, trigger it and look at your Google Cloud Function log. You'll see a stacktrace of your error, e.g.
+Finally, trigger it and look at your Google Cloud Function log. You'll see a stacktrace of your error, for example:
 
 > ```
 > ReferenceError: abc is not defined at helloWorld (/user_code/index.js:20:14)
@@ -24,7 +24,7 @@ From that stacktrace, see that in our example the wrapper running your Google Cl
 
 ## Contents of `worker.js`
 
-Although `fs.readFileSync` is not available in the given node js environment (i.e. `var fs = require("fs")` will fail) but we can use the Go runtime to give us the file contents
+Although `fs.readFileSync` is not available in the given node js environment (i.e. `var fs = require("fs")` will fail) but we can use the Go runtime to give us the file contents:
 
 > ``` go
 > func main() {

```